### ISSUE TITLE
EPUB format instead of KEPUB

### ISSUE THREAD

**Issue by fturco** [TnS-hun/kobo-book-downloader#11]
As far as I know `kobo-book-downloader` currently downloads Kobo e-books in the proprietary KEPUB format only. Is it possible to let the user download e-books in the standard EPUB format instead? Is Adobe Digital Editions the only way to accomplish that?

**Comment by jmvermeulen** [comment]
Yes that's possible, the epub variant is protected by Social DRM as Kobo calls it. The file is watermarked with a identifier to the downloader.

**Comment by fturco** [comment]
Great, but how can I get them?

**Comment by subdavis** [comment]
This repository has been abandoned. I have forked the project at https://github.com/subdavis/kobo-book-downloader - please give that project a try and open an issue there if you have trouble.

It adds some new features:

* audiobook support
* multi-user support; fetch books for multiple accounts.
* web interface; adds new browser gui (with flask)
* [docker image](https://hub.docker.com/r/subdavis/kobodl)
* [pypi package](https://pypi.org/project/kobodl/)

**Please don't reply to this issue, open an issue on my fork instead**
